xml_inscriptions/transcribed/MA.SH.MHC.AM.G.1920.4.C.F-mdn.xml

```diff
@@ -292,34 +298,38 @@
         
         <surface>
             <desc/>
-            <graphic url="MA.SH.MHC.AM.L.1920.4.C.F.jpg"/>
-        </surface>
-        <surface>
-            <desc/>
-            <graphic url="MA.SH.MHC.AM.L.1920.4.C.F_edit.jpg"/>
+            <graphic url=" MA.SH.MHC.AM.L.1920.4.C.F-mdn.jpg"/>
         </surface>
+        
     </facsimile>
 
     <text>
         
         <body>
         <div type="edition">
                 <ab>
-                    <lb n="1"/><g type="chi-rho">☧</g>
+                    <lb n="1"/><expan><abbr><hi rend="ligature">Χρ</hi></abbr><ex>ιστός</ex></expan>.
```

Comment:
So, there's a bit of a question about how to handle the monogram in the EpiDoc community generally; my sense is that we don't normally treat it as an abbreviation, or if you do, you maintain the Christogram, especially since there's a unicode character for it. (So, I'm not convinced an expansion here is necessary, since the Christogram has a specific semantic meaning. If there's a broader decision that's being made at the project level I'd like to know about it, naturally; my instinct is not to treat this as a ligatured abbreviation, but I welcome being corrected.)